### ISSUE TITLE
Simply Minesweeper (DSiWare)

### ISSUE THREAD

**Issue by milesthecreator** [Olivercomet/EPFExplorer#20]
Greetings. This game appears to use all files that are .arc, .rdt and .bin, figured I would use this tool but it doesn't seem to work properly. (e.g. some rdt files won't open at all, broken sound files)

Would it be possible to add support for this game?

**Comment by Olivercomet** [comment]
I've had a look, but it's been so long that  I wouldn't be able to implement support without investing a lot of time into it; more time than I have at the moment. For what it's worth, mp2.bin seemingly contains just one big sample; no sequenced data like the other games.

**Comment by milesthecreator** [comment]
Yeah. This game's music is streamed, couldn't find any way of getting it extracted unfortunately. Same with sound effects.

In case you'll look into it someday, take your time and good luck!
